ts/features/messages/components/Home/MessageList.tsx

```diff
@@ -122,11 +121,7 @@ export const MessageList = React.forwardRef<FlatList, MessageListProps>(
         ListEmptyComponent={<EmptyList category={category} />}
         ItemSeparatorComponent={messageList ? () => <Divider /> : undefined}
         ListHeaderComponent={
-          category === "INBOX" ? (
-            <ItwDiscoveryBanner
-              fallbackComponent={<SettingsDiscoveryBanner />}
-            />
-          ) : undefined
+          category === "INBOX" ? <LandingScreenBannerPicker /> : undefined
```

Comment:
Let's add a `__mock__/LandingScreenBannerPicker.tsx` under the components parent folder so any reference in this component uses a mocked one. We can use `ts/features/messages/components/Home/__mocks__/Preconditions.tsx` as a reference for how to implement it